libraries/AP_Networking/AP_Networking_CAN.cpp

```diff
@@ -180,18 +180,29 @@ void AP_Networking_CAN::mcast_server(void)
             */
             AP_HAL::CANFrame frame;
             const uint16_t timeout_us = 2000;
+#if AP_NETWORKING_CAN_MCAST_BRIDGING_ENABLED
+            const bool bridged = AP::network().is_can_mcast_ep_bridged(bus);
+#else
+            const bool bridged = false;
+#endif
 
             while (frame_buffers[bus]->peek(frame)) {
                 auto *cbus = get_caniface(bus);
                 if (cbus == nullptr) {
                     break;
                 }
-                auto retcode = cbus->send(frame,
-                                                      AP_HAL::micros64() + timeout_us,
-                                                      AP_HAL::CANIface::IsMAVCAN);
-                if (retcode == 0) {
-                    break;
+                if (bridged) {
+                    auto retcode = cbus->send(frame, AP_HAL::micros64() + timeout_us,
```

Comment:
This is dead code on bootloader... I wonder if the compiler is smart enough to elide..
.

Reply:
probably, but going to play it safe and forcing it out.